app/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -318,4 +324,54 @@ describe("Summary", () => {
       expect(errorContainer).toHaveTextContent(errorMessage);
     });
   });
+
+  it("handles dirty Live to Review as expected", async () => {
+    const { mockRollout, rollout } = mockLiveRolloutQuery("demo-slug", {
+      status: NimbusExperimentStatusEnum.LIVE,
+      publishStatus: NimbusExperimentPublishStatusEnum.DIRTY,
+      statusNext: null,
+    });
+
+    const mutationMock = createMutationMock(
+      rollout.id!,
+      NimbusExperimentPublishStatusEnum.DIRTY,
+      {
+        changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW_UPDATE,
+        statusNext: null,
+        publishStatus: NimbusExperimentPublishStatusEnum.DIRTY,
+        status: NimbusExperimentStatusEnum.LIVE,
+      },
+    );
+    render(<Subject props={rollout} mocks={[mockRollout, mutationMock]} />);
+
+    const requestUpdateButton = await screen.findByTestId(
+      "request-live-update-alert",
+    );
+    expect(requestUpdateButton).toBeEnabled();
+    await act(async () => void fireEvent.click(requestUpdateButton));
+  });
+
+  it("shows update and end buttons for live dirty rollout", async () => {
```

Comment:
@yashikakhurana what do you think?

Reply:
yup 💯 , thank you @eliserichards

Reply:
and now when we will change the name from `end experiment` to `end rollout`, your super test will be able to catch this 👏